charts/images.yaml

```diff
@@ -142,10 +142,28 @@ images:
   sourceRepository: github.com/coredns/coredns
   repository: coredns/coredns
   tag: "1.10.0"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'protected'
+      authentication_enforced: false
+      user_interaction: 'end-user'
+      confidentiality_requirement: 'low'
+      integrity_requirement: 'high'
+      availability_requirement: 'high'
 - name: node-local-dns
   sourceRepository: github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns
   repository: registry.k8s.io/dns/k8s-dns-node-cache
   tag: "1.22.15"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'private'
```

Comment:
As node-local-dns also talks to the infrastructure DNS, would 'protected' the right categorisation. 
Shouldn't it have the same value as coredns?

Reply:
`coredns` talks to `kube-apiserver`, which is not located in the same VPC, while `nodelocaldns` does not.
Whether you consider the infrastructure DNS to be within the same VPC or not is a discussion on its own.
I can change the categorisation though.
WDYT?

Reply:
I didn't think about the api-server. Then, it's OK like this.